beetsplug/lyrics.py

```diff
@@ -459,12 +524,16 @@ def __init__(self):
             'auto': True,
             'google_API_key': None,
             'google_engine_ID': u'009217259823014548361:lndtuqkycfu',
+            'genius_api_key':
+                "Ryq93pUGm8bM6eUWwD_M3NOFFDAtp2yEE7W"
+                "76V-uFL5jks5dNvcGCdarqFjDhP9c",
```

Comment:
this is the api auth key for the app, which is now registered with genius.  let me know if there's a better place to put this; i'm not crazy about inlining it but since the lyrics plugin is not enabled by default it wouldn't make any sense to put it in ```config_default.yaml```.